x/btcstkconsumer/genesis.go

```diff
@@ -0,0 +1,23 @@
+package btcstkconsumer
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/babylonlabs-io/babylon/x/btcstkconsumer/keeper"
+	"github.com/babylonlabs-io/babylon/x/btcstkconsumer/types"
+)
+
+// InitGenesis initializes the module's state from a provided genesis state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
+	if err := k.SetParams(ctx, genState.Params); err != nil {
+		panic(err)
+	}
+}
+
+// ExportGenesis returns the module's exported genesis.
+func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
```

Comment:
Same, genesis state should have every data needed to relaunch the chain in case of failure

Reply:
Sg, covered in https://github.com/babylonlabs-io/babylon/issues/426